tests/DCPS/Thrasher/DataReaderListenerImpl.cpp

```diff
@@ -69,10 +69,12 @@ void DataReaderListenerImpl::on_sample_lost(DDS::DataReader_ptr,
 
 bool DataReaderListenerImpl::wait_received(const OpenDDS::DCPS::TimeDuration& duration, size_t target)
 {
+  ACE_DEBUG((LM_INFO, ACE_TEXT("%T(%t) sub->wait received_samples %d\n"), received_samples_));
```

Comment:
%T is unnecessary because the test can be configured with OrbVerboseLogging.  If you are going to add %t, then you should probably have %P|%t.

Reply:
Please add the OrbVerboseLogging to this test as that will help in the future.